src/screens/QuestionsScreen.tsx

```diff
@@ -20,15 +20,13 @@ type QuestionScreenProps = {
   onContinue: () => void
 }
 
-const headphoneOptions = [{ label: '', value: undefined }].concat(
-  [
-    "I hadn't heard any loud noises",
-    'After I heard one loud noise',
-    'After I heard a few loud noises',
-    'After I reached a break',
-    'During questionnaires',
-  ].map((q) => ({ label: q, value: q })),
-)
+const headphoneOptions = [
+  "I hadn't heard any loud noises",
+  'After I heard one loud noise',
+  'After I heard a few loud noises',
+  'After I reached a break',
+  'During questionnaires',
+].map((q) => ({ label: q, value: q }))
```

Comment:
**Possible Issue:** Something in the back off my head is telling me the values here should be slugified? but I guess it doesn't matter if they are not being used as url params?

Reply:
Yes, we want the actual result for this one, bit weird I know. It's just submitted as a json string.